build.sbt

```diff
@@ -59,8 +59,8 @@ lazy val plugin = (project in file("."))
     libraryDependencies ++= List(
       "dev.gradleplugins" % "gradle-api" % V.gradle % Provided,
       "dev.gradleplugins" % "gradle-test-kit" % V.gradle % Provided,
-      "org.gradle" % "gradle-core" % V.gradle % Provided,
-      "org.gradle" % "gradle-tooling-api" % V.gradle % Provided,
+      //   "org.gradle" % "gradle-core" % V.gradle % Provided,
```

Comment:
As I understand, the `gradle-bloop` depends on it. Will this change break the project, or I miss something?